.github/workflows/sbom-generator.yaml

```diff
@@ -23,7 +26,30 @@ jobs:
         id: sbom
         env: 
           GITHUB_TOKEN: ${{ github.token }}
+
       - uses: actions/upload-artifact@v3.1.0
         with: 
           path: ${{steps.sbom.outputs.fileName }}
           name: "SBOM"
+
+      - name: Get latest release
+        id: latest_release
+        uses: actions/github-script@v4
+        with:
+          script: |
+            const { data } = await github.repos.listReleases({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              per_page: 1
+            });
+            return data[0];
+
+      - name: Upload SBOM as release asset
+        uses: actions/upload-release-asset@v1
```

Comment:
Cool find! you can execute this step under the condition that I mentioned in my comment above.